docs/processing_data.rst

```diff
@@ -152,6 +173,235 @@ Although this is a simple example of labeling data other things can be done here
+
++----------------------+
+| |flow_southcontrols| |
++----------------------+
+
+  - **Service Status** - a coloured indicator of the current monitored status:
+    
+      - Green indicates the service is running and the pipeline is processing data.
+
+      - Yellow indicates the service has started to show signs of failure.
+
+      - Red indicates the service has failed.
+
+      - Grey indicates the service is not enabled.
+
+   - **Enable Control** - a toggle control that can be used to enable and disable the service.
+
+   - **Configuration** - a control that can be used to display the configuration dialog for the plugin. This allows the configuration to be changed for an existing south plugin.
+
+   - **Menu** - enable the display of the context menu for the service:
+
+     +------------------+
+     | |flow_southmenu| |
+     +------------------+
+
+     - Readings - display the readings that are buffered in the storage service as a result of this pipeline.
+
+     - Export Readings - export the buffered readings for this pipeline to a CSV file.
+
+     - Delete - delete the service.
+
+  - **Display Readings** - display the readings that are buffered in the storage service as a result of this pipeline.
+
+  - **Show Logs** - display the logs written by this service.
+
+  - **Help** - show the online documentation of this south plugin. The online documentation will be shown in a new browser tab.
+
+These same controls and status indicators are also available in the south page that shows all of the current south services that are configured within the Fledge instance.
+
++--------------+
+| |flow_south| |
++--------------+
+
+Adding Filters
+~~~~~~~~~~~~~~
+
+Adding a new filter to a pipeline is a simple process of dragging the blank filter icon from the bottom left of the canvas and dropping it on one of the connecting lines in the filter pipeline.
+
++------------------+
+| |flow_addfilter| |
++------------------+
+
+Once the new filter has been dropped onto the connection, the pipeline will redraw and show an empty filter in the pipeline.
+
++--------------------+
+| |flow_filteradded| |
++--------------------+
+
+The dashed outline of the filter signifies that the filter has yet to be defined, its place within the pipeline is set, but the actual filter to be inserted as not been selected and it has not been configured. Clicking on the + icon will bring up the filter selection dialog.
+
++---------------------+
+| |flow_definefilter| |
++---------------------+
+
+Select the filter plugin to use from the list of plugins given. If a filter you need has not been installed you may install it by clicking on the *Install from available plugins* link below the list.
+
+Give the filter a name, names must be unique within the Fledge instance. Once complete click on the *Next* button to configure your chosen filter plugin.
```

Comment:
Give the filter a name, names must be unique -> Give the filter a name. Names must be unique